grails-app/services/org/pih/warehouse/data/LoadDataService.groovy

```diff
@@ -200,52 +209,120 @@ class LoadDataService {
         csvReader.close();
     }
 
+    private Transaction createAdjustmentTransaction(Date transactionDate, Inventory inventory) {
+        return new Transaction(
+                transactionDate: transactionDate,
+                transactionType: TransactionType.get(Constants.ADJUSTMENT_CREDIT_TRANSACTION_TYPE_ID),
+                inventory: inventory
+        )
```

Comment:
how is this working without setting a transactionNumber?

Reply:
I checked the transactionNumber and it seems to be autogenerated :thinking: 
![image](https://github.com/user-attachments/assets/4fc3c082-7e16-4abe-8777-7482f2ee4b02)